### PR TITLE
Accessing minikube Dashboard UI

### PR DESCRIPTION
**Pull request by AbdullahBM**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -109,6 +109,10 @@ Start a cluster by running:
 
 `minikube start`
 
+Access Minikube Dashboard UI:
```
Comment (tstromberg) [inline README.md]: technically, this is a Kubernetes dashboard. How about:

"Access the Kubernetes dashboard UI"?

Comment (AbdullahBM) [reply README.md]: For accessing Kubernetes Web UI (Dashboard), there is another process, in which we first install Kubernetes UI and then access it. While Minikube dashboard is easy to start. Just:
"minikube dashboard"

Comment (tstromberg) [reply README.md]: This command is just a short-cut to start the Kubernetes dashboard UI inside of minikube.

Do you mind changing the text to say "Access the Kubernetes dashboard within minikube"?

Thanks!

Comment (AbdullahBM) [reply README.md]: Yes. You are right. It is basically a Kubernetes Dashboard Addon.

Comment (RA489) [reply README.md]: @AbdullahBM please address the changes @tstromberg  suggested rest lgtm.

Comment (AbdullahBM) [reply README.md]: Done. Please check it @tstromberg.

**Pull request merged**